tests/acceptance/DownloadGigaDBTemplateFiles.feature

```diff
@@ -0,0 +1,22 @@
+Feature: An author can download file template
+  As an author
+  I want to download upload form template after login
+  So that I can upload data using the form
```

Comment:
I think something like this will be more accurate:
```
I want to download the dataset metadata upload form template after login
So that I can upload information about my dataset using the form
```